### PR TITLE
Updated spacing in checkmark

### PR DESCRIPTION
**Pull request by hemanthReddy06**

### CODE REVIEW

File: dimagi/utils/pdf/pricing.py [hunk 1]
```diff
@@ -395,4 +406,4 @@ def get_pricing_pdf(groups, is_monthly=False):
     ))
 
     doc.build(story)
-    return data
+    return data
```
Comment (biyeun) [inline dimagi/utils/pdf/pricing.py]: please add a blank new line at the end of file? why was this removed?

Comment (hemanthReddy06) [reply dimagi/utils/pdf/pricing.py]: Editor mistake.
